src/App.tsx

```diff
@@ -1,26 +1,160 @@
-/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { USER_ID } from './api/todos';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+import { createTodo } from './api/todos';
+import { delTodos } from './api/todos';
+import { TodoHeader } from './components/TodoHeader';
+import { TodoList } from './components/TodoList';
+import { TodoFooter } from './components/TodoFooter';
+import { ErrorNotification } from './components/ErrorNotification';
+export const App: React.FC = () => {
+  const [isInput, setIsInput] = useState('');
+  const [isTodo, setTodo] = useState<Todo[]>([]);
+  const [filter, setFilter] = useState<'all' | 'active' | 'completed'>('all');
+  const [errorType, setErrorType] = useState<string | null>(null);
+  const [tempTodo, setTempTodo] = useState<null | Todo>(null);
+  const [isLoading, setIsLoading] = useState(false);
+  const [loadingTodoId, setLoadingTodoId] = useState<number[]>([]);
 
-const USER_ID = 0;
+  const handleError = (type: string) => {
+    setErrorType(type);
+    setTimeout(() => setErrorType(null), 3000);
+  };
 
-export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
+  useEffect(() => {
+    if (!USER_ID) {
+      return <UserWarning />;
+    }
+
+    getTodos()
+      .then(data => setTodo(Array.isArray(data) ? data : []))
+      .catch(() => {
+        handleError('Unable to load todos');
+      })
+      .finally(() => {});
+  }, []);
+
+  function addTodo(userId, title, completed) {
```

Comment:
It is important to stick to one way of creating functions, fix it everywhere
```suggestion
  const addTodo = (userId, title, completed) => {
```